dll/win32/shell32/lang/ja-JP.rc

```diff
@@ -140,22 +140,22 @@ BEGIN
     LTEXT COPYRIGHT_YEAR, IDC_STATIC, 88, 75, 17, 10
     LTEXT " ReactOS Team\0", IDC_STATIC, 105, 75, 53, 10
     LTEXT "", IDC_ABOUT_OTHERSTUFF, 35, 90, 180, 20
-    LTEXT "このReactOSのバージョンは次の方に登録されています:", IDC_STATIC, 35, 115, 180, 10
-    LTEXT "", IDC_ABOUT_REG_USERNAME, 45, 125, 180, 10
-    LTEXT "", IDC_ABOUT_REG_ORGNAME, 45, 135, 180, 10
-    LTEXT "", IDC_STATIC, 35, 147, 235, 1, SS_ETCHEDHORZ
-    LTEXT "搭載されている物理メモリ容量:", IDC_STATIC, 35, 152, 130, 10
-    LTEXT "", IDC_ABOUT_PHYSMEM, 167, 152, 88, 10
-    DEFPUSHBUTTON "OK", IDOK, 220, 178, 50, 14
-    PUSHBUTTON "", IDC_ABOUT_AUTHORS, 5, 178, 70, 14
+    LTEXT "このReactOSのバージョンは次の方に登録されています:", IDC_STATIC, 35, 105, 180, 10
+    LTEXT "", IDC_ABOUT_REG_USERNAME, 45, 115, 180, 10
+    LTEXT "", IDC_ABOUT_REG_ORGNAME, 45, 125, 180, 10
+    LTEXT "", IDC_STATIC, 35, 125, 235, 1, SS_ETCHEDHORZ
+    LTEXT "搭載されている物理メモリ容量:", IDC_STATIC, 35, 135, 130, 10
+    LTEXT "", IDC_ABOUT_PHYSMEM, 167, 135, 88, 10
+    DEFPUSHBUTTON "OK", IDOK, 167, 158, 50, 14
+    PUSHBUTTON "", IDC_ABOUT_AUTHORS, 5, 158, 50, 14
 END
 
 IDD_ABOUT_AUTHORS DIALOGEX 35, 90, 235, 85
 STYLE DS_SHELLFONT | WS_CHILD
 FONT 9, "MS UI Gothic"
 BEGIN
-    LTEXT "ReactOS は次の方々によって提供されています:", IDC_STATIC, 0, 0, 180, 10
-    LISTBOX IDC_ABOUT_AUTHORS_LISTBOX, 0, 10, 165, 75
+    LTEXT "ReactOSは次の方々によって提供されています:", IDC_STATIC, 0, 0, 180, 10
```

Comment:
What happened to the space after "ReactOS"?

Reply:
I don't know how it's called, but it is normal for english words in japanese sentences.

Reply:
OK.